### PR TITLE
Update to Phoenix 1.2 and Postgres

### LINKED ISSUE #79: Update to Phoenix 1.2 with Postgres
Migrate app to use Elixir 1.2, Phoenix 1.2, and switch to PostgreSQL.

### PR DESCRIPTION
**Pull request by troym9731**
Closes #79

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -11,7 +11,8 @@ config :org, Org.Endpoint,
   debug_errors: true,
   code_reloader: true,
   check_origin: false,
-  watchers: [node: ["node_modules/brunch/bin/brunch", "watch", "--stdin"]]
+	watchers: [node: ["node_modules/brunch/bin/brunch", "watch", "--stdin",
```
Comment (therealklanni) [inline config/dev.exs]: Not sure if it's just the way GitHub is displaying it, but it _looks_ like there's an extra space in the indent.

File: test/support/channel_case.ex [hunk 2]
```diff
@@ -32,9 +32,11 @@ defmodule Org.ChannelCase do
   end
 
   setup tags do
-    unless tags[:async] do
-      Mongo.Ecto.truncate(Org.Repo, [])
-    end
+		:ok = Ecto.Adapters.SQL.Sandbox.checkout(Org.Repo)
```
Comment (therealklanni) [inline test/support/channel_case.ex]: Indentation looks wrong here as well.

File: test/support/model_case.ex [hunk 3]
```diff
@@ -20,25 +20,27 @@ defmodule Org.ModelCase do
...
+      import Ecto.Query
       import Org.ModelCase
     end
   end
 
   setup tags do
-    unless tags[:async] do
-      Mongo.Ecto.truncate(Org.Repo, [])
-    end
+		:ok = Ecto.Adapters.SQL.Sandbox.checkout(Org.Repo)
```
Comment (therealklanni) [inline test/support/model_case.ex]: And here

**Review by therealklanni — APPROVED** [review]
:shipit:

**Comment by therealklanni** [comment]
The only thing that concerns me is the fact that secret and salt have been regenerated. Do you think this will affects users being able to log in?

**Comment by troym9731** [comment]
I'm unsure. I'll just revert it back to the old ones.

**Comment by troym9731** [comment]
Only took 4 commits but I fixed all the issues 😅

**Pull request merged**